osu.Game.Rulesets.Taiko/Difficulty/TaikoPerformanceCalculator.cs

```diff
@@ -61,31 +61,40 @@ protected override PerformanceAttributes CreatePerformanceAttributes(ScoreInfo s
 
         private double computeDifficultyValue(ScoreInfo score, TaikoDifficultyAttributes attributes)
         {
-            double difficultyValue = Math.Pow(5 * Math.Max(1.0, attributes.StarRating / 0.175) - 4.0, 2.25) / 450.0;
+            double difficultyValue = Math.Pow(5 * Math.Max(1.0, attributes.StarRating / 0.115) - 4.0, 2.25) / 1150.0;
```

Comment:
This doesn't seem trivial to understand intuitively, would be nice to add some sort of documentation to this in the future

Reply:
For now I've mapped it on [desmos](https://www.desmos.com/calculator/y3tkynmiqj) to understand its linear scaling.